From fb792883f3ccc55997fdc21a9c1052f778dea1ac Mon Sep 17 00:00:00 2001
From: rpm-build <rpm-build>
Date: Wed, 6 Mar 2024 19:17:14 +0100
Subject: [PATCH 01/50] RH: Aarch64 and ppc64le use lib64

Patch-name: 0001-Aarch64-and-ppc64le-use-lib64.patch
Patch-id: 1
Patch-status: |
    # # Patches exported from source git
    # # Aarch64 and ppc64le use lib64
From-dist-git-commit: 4334bc837fbc64d14890fdc51679a80770d498ce
---
 Configurations/10-main.conf | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Configurations/10-main.conf b/Configurations/10-main.conf
index cba57b4127..3e327017ef 100644
--- a/Configurations/10-main.conf
+++ b/Configurations/10-main.conf
@@ -726,6 +726,7 @@ my %targets = (
         lib_cppflags     => add("-DL_ENDIAN"),
         asm_arch         => 'ppc64',
         perlasm_scheme   => "linux64le",
+        multilib         => "64",
     },
 
     "linux-armv4" => {
@@ -768,6 +769,7 @@ my %targets = (
         inherit_from     => [ "linux-generic64" ],
         asm_arch         => 'aarch64',
         perlasm_scheme   => "linux64",
+        multilib         => "64",
     },
     "linux-arm64ilp32" => {  # https://wiki.linaro.org/Platform/arm64-ilp32
         inherit_from     => [ "linux-generic32" ],
-- 
2.49.0

